README.md

```diff
@@ -1,24 +1,49 @@
+[![Travis CI Build Status](https://travis-ci.org/mozilla/rkv.svg?branch=master)](https://travis-ci.org/mozilla/rkv)
+[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/lk936u5y5bi6qafb/branch/master?svg=true)](https://ci.appveyor.com/project/mykmelez/rkv/branch/master)
+[![Documentation](https://docs.rs/rkv/badge.svg)](https://docs.rs/rkv/)
+[![Crate](https://img.shields.io/crates/v/rkv.svg)](https://crates.io/crates/rkv)
+
 # rkv
 
-<a href="https://crates.io/crates/rkv">
-    <img src="https://img.shields.io/crates/v/rkv.svg">
-</a>
+The [rkv Rust crate](https://crates.io/crates/rkv) is a simple, humane, typed Rust interface to [LMDB](http://www.lmdb.tech/doc/).
+
+## Use
+
+Comprehensive information about using rkv is available in its [online documentation](https://docs.rs/rkv/), which you can also generate for local consumption:
 
-rkv is a usable Rust wrapper around LMDB.
+```sh
+cargo doc --open
```

Comment:
perhaps `cargo doc --no-deps --open`?

Reply:
Nvm, just notice that there are some cross references on deps' doc in the crate document.

Reply:
Yes, my intent is to reference dependencies in the rkv docs, and to ensure that I'm referencing the correct versions of those dependencies (i.e. the versions that rkv is actually dependent on, rather than the latest versions available on crates.io), I reference the docs created by the `cargo doc` invocation that created the rkv docs.